src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -49,6 +67,71 @@ def get_osd_map(self):
             osds[str(osd['id'])] = osd
         return osds
 
+    def _convert_expectency_state(self, life_expectancy_weeks):
```

Comment:
I agree it would be nice if these thresholds were configurable. I wonder if they should map to the thresholds that are used to either raise a health warning or mark the affected OSD as out?

Reply:
@LenzGr Does your mean like as the pull request https://github.com/ceph/ceph/pull/21301?

Reply:
No, I was more referring to the values for `mark_out_threshold` and `warn_threshold` in #22692. As [John pointed out](https://github.com/ceph/ceph/pull/22785#issuecomment-402205018), it might be confusing to work with two different thresholds here.